### ISSUE TITLE
Which part of the dataset is for validation

### ISSUE THREAD

**Issue by Shentl** [bupt-ai-cz/BCI#9]
In paper "Breast Cancer Immunohistochemical Image Generation: a Benchmark Dataset and Challenge Review", it is menntioned that "The dataset used in this challenge consists of 3396 pairs of the training set images, 500 pairs of the validation set images", but I can't  find any information for the val set both on the official website https://bci.grand-challenge.org/ and the dataset

        So, which part in the train fold is for validation? Or just split the dataset randomly
